src/spec.c

```diff
@@ -992,7 +992,10 @@ char *IndexSpec_GetRandomTerm(IndexSpec *sp, size_t sampleSize) {
 
 ///////////////////////////////////////////////////////////////////////////////////////////////
 
-void IndexSpec_FreeInternals(IndexSpec *spec) {
+
+// unlink spec from global structures:
+// specs, schema prefixes, aliases, cursors, and GC 
+void IndexSpec_FreeGlobals(IndexSpec *spec) {
   if (spec->name && dictFetchValue(specDict_g, spec->name) == spec) {
```

Comment:
So this is still looking at `specDict_g` which means that we did not fix the swapdb bug?

Reply:
The swapdb uses the `Indexes_Free` function which does not call `IndexSpec_FreeGlobals`